src/test_sum_series.py

```diff
@@ -0,0 +1,22 @@
+"""Test  file for sum series."""
+import pytest
```

Comment:
you don't actually need this import unless you are going to use something defined by pytest, like `pytest.raises` or `pytest.fixture` or `pytest.mark.parametrize`.